Makefile

```diff
@@ -391,6 +393,7 @@ $(FCITX5_QT): | deps
 PRODUCTION_PARAMETERS := -d:production -d:chronicles_sinks=textlines[stdout],textlines[nocolors,dynamic]
 
 $(STATUS_CLIENT_APPIMAGE): override RESOURCES_LAYOUT := $(PRODUCTION_PARAMETERS)
+$(STATUS_CLIENT_APPIMAGE): IPFS_GATEWAY_URL_ARG = $(PROD_IPFS_GATEWAY_URL_ARG)
```

Comment:
Why are you using the prod endpoint for non-release builds?

Reply:
As far as I understand, those are when we do `make pkg`, so it's what we do when we create a release build? Which command would be the one that is used for release otherwise?
I followed the places where we use `PRODUCTION_PARAMETERS`

Reply:
The way you are doing it it will affect all builds. The correct way is detecting a release build in CI. Like I do here:

* https://github.com/status-im/status-jenkins-lib/pull/44
* https://github.com/status-im/status-mobile/pull/13850

I use [`utils.isReleaseBuild()`](https://github.com/status-im/status-jenkins-lib/blob/db0161b8d79493f7d7b14f8d736e2b162f8244d8/vars/utils.groovy#L151-L159) to provide a different URL for actual release builds.

Reply:
Unless your intention isn't to actually use that URL for actual release builds but for CI builds as a whole.

Can you clarify the intention? I assumed that we want to distinguish between actual release builds like in mobile.

Reply:
I think the goal would be mostly for release builds, but I don't know how to differentiate a normal package build vs a release build. 
We have `RELEASE ?= false`, but it never gets set to `true`.

Anyway, since this should be a temporary solution, I think using the "prod" url on all builds makes sense, since QAs use those builds to test the app and if we ever need a build to pass to a test user, we'd want the prod url as well I think.

What do you think?

Reply:
Honestly, none of this `IPFS_GATEWAY_URL_ARG` is necessary at all since you've used `?=` assignment in your PR.

That means we can entirely drop any of the `Makefile` changes here, and just do it in `status-jenkins-lib`, like this:
https://github.com/status-im/status-jenkins-lib/commit/d4e69880
I made that change part of my PR intended initially for mobile:

* https://github.com/status-im/status-jenkins-lib/pull/44

Reply:
Ah cool, so as I understand, with that change you did here https://github.com/status-im/status-jenkins-lib/commit/d4e69880, I can drop the changes in the Makefle?

Reply:
Correct, this way the only builds that will be affected by a different URL are actually release builds based on the `utils.isReleaseBuild()` check.

Reply:
Updated. No more Makefile changes

Reply:
But you are not using the changes in `status-jenkins-lib`.

Reply:
Updated